components/ipfs/ipfs_utils.cc

```diff
@@ -582,5 +582,4 @@ absl::optional<GURL> ExtractSourceFromGateway(const GURL& url) {
 
   return ExtractSourceFromGatewayPath(url);
 }
-
```

Comment:
maybe we can remove that, so the file doesn't get changed?

Reply:
Ah my editor removed the last line. Removed.